components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/functions/application/ServiceProviderToApiModel.java

```diff
@@ -102,6 +102,7 @@ public class ServiceProviderToApiModel implements Function<ServiceProvider, Appl
     private static final Set<String> systemApplications = ApplicationManagementServiceHolder
             .getApplicationManagementService().getSystemApplications();
     private static final String IS_FRAGMENT_APP = "isFragmentApp";
+    private static final String APPLICATION_ACCESS_ENABLED = "IsApplicationAccessEnabled";
```

Comment:
Better if we can have the similar name in API and backend service. `IsApplicationEnabled` WDYT?

Reply:
Need to change the Framework then.

Reply:
Merged the Integration test PR with this name as well.